WTraining/app/models/user.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const bcrypt = require('bcryptjs');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        is: {
+          args: [/^[A-Za-z]+$/],
```

Comment:
do not use magical values  I think that you could use a constant folder and this to set this values